package.json

```diff
@@ -101,7 +101,7 @@
     "@dnd-kit/utilities": "^2.0.0",
     "@elastic/apm-rum": "^5.16.0",
     "@elastic/apm-rum-react": "^2.0.2",
-    "@elastic/charts": "64.0.0",
+    "@elastic/charts": "64.0.1",
```

Comment:
let's move to 64.0.2  ;) that has the fix about the overlapping ticks

Reply:
Yup 👍🏼 d40842c805375a9af5cf744607a834094d2d934b